README.md

```diff
@@ -30,16 +30,21 @@ $ export CLIENT_SECRET=...
 
 Documentation
 -------------
-The documentation for the latest release can be found on [readthedocs](http://descartes-labs-python.readthedocs.io/)
+The documentation for the latest release can be found at [https://docs.descarteslabs.com](https://docs.descarteslabs.com)
 
 Changelog
 =========
 
 ## [Unreleased]
-## [0.8.0] - 2018-02-08
+## [0.8.0] - 2018-03-29
 ### Changed
-- Removed deprecated searching by const_id
+- Removed deprecated searching by `const_id`
```

Comment:
Did this happen in a prior release, the code doesn't seem to be in this change set?

Reply:
not released but it was already merged to master https://github.com/descarteslabs/descarteslabs-python/commit/274897f8f3c4d0b4ab0591287330af3b35cc3bcc